pkg/cmd/server/start.go

```diff
@@ -301,11 +326,17 @@ func start(cfg *config, args []string) error {
 			osmaster.DeployerOSClientConfig = osClientConfig
 		}
 
+		// TODO: make anonymous auth optional
```

Comment:
You don't think this ends up being optional based on whether or not policy allows anonymous access?  If a cluster-admin wanted to shut it down, all he'd have to is create a deny all policy and bind AllAnonymous to it.